python/sglang/srt/managers/tokenizer_manager.py

```diff
@@ -451,6 +461,80 @@ async def update_weights(
         else:
             return False, "Another update is in progress. Please try again later."
 
+    async def init_parameter_update_group(
+        self,
+        obj: InitParameterUpdateGroupReqInput,
+        request: Optional[fastapi.Request] = None,
+    ) -> bool:
+        if self.to_create_loop:
+            self.create_handle_loop()
+
+        if obj.backend is None:
+            obj.backend = "nccl"
+        self.send_to_scheduler.send_pyobj(obj)
+
+        self.init_parameter_update_group_result = asyncio.Future()
+
+        if self.server_args.dp_size == 1:
```

Comment:
it seems this only support dp_size = 1, so you can `assert self.server_args.dp_size == 1` here